DFC.App.MatchSkills.Test/Unit/Controllers/OccupationSearchControllerTests.cs

```diff
@@ -67,19 +67,54 @@ public void  When_OccupationSearch_Then_ShouldReturnOccupations()
         [Test]
         public void  When_OccupationSearchAuto_Then_ShouldReturnOccupations()
         {
-            var sut = new OccupationSearchController(serviceTaxonomyRepository,_settings,_compositeSettings, _sessionService, _cookieService);
+            var sut = new OccupationSearchController(_serviceTaxonomyRepository,_settings,_compositeSettings, _sessionService, _cookieService);
             
             var occupations =   sut.OccupationSearchAuto("Renewable");
             
             occupations.Result.Should().NotBeNull();
         }
 
-       
+        [Test]
+        public void  When_GetOccupationIdFromName_Then_ShouldReturnOccupationId()
+        {
+            const string skillsJson ="{\"occupations\": [{\"uri\": \"http://data.europa.eu/esco/occupation/114e1eff-215e-47df-8e10-45a5b72f8197\",\"occupation\": \"Renewable energy consultant\",\"alternativeLabels\": [\"alt 1\"],\"lastModified\": \"03-12-2019 00:00:01\"}]}";           
```

Comment:
Seems like we're using the const, multiple times, can we instantiate it at the top of the class instead please?

Reply:
Done